<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="powerOnBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Power ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="powerOffBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Power OFF</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>241</width>
      <height>281</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Test Mode</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QListWidget" name="listWidget">
       <item>
        <property name="text">
         <string>=================</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TXSTART</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TXDATA1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TXDATA2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TXDATA3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TXDATA4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>=================</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RXSTART1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RXSTART2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RXDATA1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RXDATA2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BIT ERR1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>=================</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CFG FREQ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CFG PKT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CFG TX IF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CFG HOPPING SEQ</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>291</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Test Arguments</string>
    </property>
    <widget class="QLineEdit" name="param1Edit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="param1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>param1</string>
     </property>
    </widget>
    <widget class="QLabel" name="param2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>param2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="param2Edit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="param3Edit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="param3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>param3</string>
     </property>
    </widget>
    <widget class="QLabel" name="param4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>param4</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="param4Edit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="excuteBtn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>100</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Excute</string>
    </property>
   </widget>
   <widget class="QPushButton" name="warmResetBtn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>160</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Warm Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>735</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
